framework/source/class/qx/core/Property.js

```diff
@@ -797,30 +811,27 @@ qx.Bootstrap.define("qx.core.Property",
      * Installs a getter into the class prototype, without executing it
      * Supported variants: get
      *
-     * @param instance {Object} the instance which calls the method
      * @param clazz {Class} the class which originally defined the property
      * @param name {String} name of the property
      * @param variant {String} Method variant.
      */
-    installOptimizedGetter : function(instance, clazz, name, variant)
+    installOptimizedGetter : function(clazz, name, variant)
```

Comment:
ah - yes, that's true; the problem is that an instance is simply not always available when installing the accessor methods, those installOptimised* methods were introduced in the patch for qxcompiler and were simply following the implementation of teh functions they were split off from.  Arguably they shouldn't be exposed in the first place, but as they never made it to a released version, I propose making them private instead.